Index.html

```diff
@@ -4,7 +4,7 @@
 <title>Page Title</title>
 </head>
 <body>
-
+    <h1>This is my first portion</h1>
```

Comment:
Could you change this and make it look more like a landing page?, "Welcome to Portionify" would have been a better choice

Reply:
i will make changes and revert

Reply:
changes is made